components/script/dom/htmlselectelement.rs

```diff
@@ -28,7 +32,8 @@ use style::element_state::*;
 
 #[dom_struct]
 pub struct HTMLSelectElement {
-    htmlelement: HTMLElement
+    htmlelement: HTMLElement,
+    options: MutNullableHeap<JS<HTMLOptionsCollection>>,
```

Comment:
Looks like this isn't possible to create an `HTMLOptionsCollection` during the creation of `HTMLSelectElement`. Ignore my comment and leave it as-is.